identity-admin/webapp/components/search/Sorter.test.tsx

```diff
@@ -0,0 +1,62 @@
+import Sorter from './Sorter';
+import { SortField } from '../../interfaces';
+import { render } from '@testing-library/react';
+import { useRouter } from 'next/router';
+import { buildSearchUrl } from '../../utils/search-util';
+
+const mockedRouter = useRouter as jest.Mock;
+
+jest.mock('next/router');
+
+const renderComponent = (sort: SortField) => {
+  return render(<Sorter fieldName={sort} />);
+};
+
+const mockQueryParams = (sortDir: string | undefined, sort: SortField) => {
+  mockedRouter.mockImplementation(() => {
+    return {
+      query: {
+        status: 'active',
+        name: 'Bob',
+        email: 'bob@email.com',
+        sort: sort,
+        sortDir: sortDir,
+      },
+    };
+  });
+};
```

Comment:
This is cool, but I think it's a lot more usable if we pass an object here:

```ts
const mockQueryParams = (params: { sortDir?: string; sort: SortField }) => {
  mockedRouter.mockImplementation(() => {
    return {
      query: {
        status: 'active',
        name: 'Bob',
        email: 'bob@email.com',
        ...params,
      },
    };
  });
};
```
(I'm not sure why `sortDir` can be `undefined` here?)

Then it's more obvious what the arguments are doing:

```ts
mockQueryParams({ sortDir: '1', sort: SortField.Email });
```

Reply:
I think 'sortDir' being undefined was a hangover from an earlier attempt to get this working - I've removed it